libs/motor/inc/public/tfc/motor/details/positioner_impl.hpp

```diff
@@ -49,6 +49,8 @@ using position_mode_config = std::variant<std::monostate,
                                           encoder_config<reference>,
                                           freq_config<deduce_velocity_t<reference>>>;
 
+enum struct limit_switch_e : std::uint8_t { not_used, limit_switches_optional };
```

Comment:
I don't quite get the point of having this.

Reply:
Do you mean on a config level or whether this should be a bool ?